idalib/examples/comments_ls.rs

```diff
@@ -35,6 +35,13 @@ fn main() -> anyhow::Result<()> {
         assert!(read_comment.unwrap().starts_with("Comment added by idalib"));
     }
 
+    println!("Testing find_text_all()");
+    let results = idb.find_text_all("added by idalib");
```

Comment:
With the proposed change to `find_text_iter`, this can be done like:

```rust
    let results = idb.find_text_iter("added by idalib").collect::<Vec<_>>();
```